guide/onboarding/creating-a-new-wallet.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: guide
+title: Creating a new wallet
+description: Steps to take when creating a new wallet
+parent: Onboarding
+nav_order: 2
+permalink: /guide/onboarding/creating-a-new-wallet/
+main_classes: -no-top-padding
+---
+
+# Creating a new wallet
+
+First, bitcoin [wallets]({{ 'guide/glossary/#wallet' | relative_url }}) are not analogous to their namesakes. Physical wallets are places to store physical currency, credit cards, IDs, and so on. Bitcoin wallets don't store bitcoin.
+
+Instead, a bitcoin wallet contains the "private keys" to your bitcoin. The bitcoin is located on the Bitcoin network that contains a public ledger similar in many ways to a spreadsheet with a record of everyone's transactions and balances. Private keys let you access bitcoin associated with those keys. A more suitable analogy for a bitcoin wallet would be a keychain.
+
+ Like regular keys on a keychain, bitcoin private keys can be copied. For example, if you gave someone a copy of your house keys, they can unlock the door to your home. Likewise, if someone has a copy of your private key, they can access your bitcoin. This makes [private key management]({{ '/guide/private-key-management/introduction' | relative_url }}) one of the most important considerations of any bitcoin owner.
+
+There are a few different types of wallets that let you send, receive, store and manage bitcoin.
+
+{% include /tip/open.html label="Tip: Break onboarding up" icon="info" color="blue" %}
+
+Wallet creation during onboarding is an excellent opportunity to educate your users. For example, using several screens when generating a  [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) to empasize it's importance. 
+
+{% include /tip/close.html %}
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/onboarding/creating-a-new-wallet/onboarding-welcome-screen.png"
+   retina = "/assets/images/guide/onboarding/creating-a-new-wallet/onboarding-welcome-screen@2x.png"
+   width = 250
+   height = 250
+   alt-text = "Onboarding welcome screen"
+   layout = "float-left-desktop -background -shadow"
+%}
+
+Most wallets will begin the onboarding process with creating a new or restoring an existing wallet. Clearly, users new to bitcoin will be going through the flow of creating a new wallet.
+
+Once an individual selects “create a new wallet”, it’s typical for beginner-friendly applications to first present informational carousels that provide a high-level overview of the implications, benefits, and responsibility that comes with having a [non-custodial]({{ '/guide/glossary/#non-custodial--custodial-wallet' | relative_url }}) wallet. This moment is a great place to explain that, unlike traditional centralized financial products, a user’s “account” information cannot be recovered by the respective company. Rather, it is up to the user to safely protect their wallet information.
```

Comment:
You use the term "non-custodial" here, was this already explained earlier? Maybe link to the glossary?

Reply:
I wonder if it needs to be clarified at the top of the page that this is about non-custodial wallets only.

Reply:
I've changed all references of non-custodial to self-custodial now I think